packages/web3-plugin-wallet-rpc/src/WalletRpcPlugin.ts

```diff
@@ -105,8 +130,9 @@ export class WalletRpcPlugin extends Web3PluginBase<WalletRpcApi> {
    *
    * See [EIP-2256](https://eips.ethereum.org/EIPS/eip-2256) for more details.
    *
-   * @param param - Details of the request for owned assets
-   * @returns a Promise that resolves to a list of owned assets
+   * @param param - Details of the request for owned assets.
+   * @returns A Promise that resolves to a list of owned assets.
+   * @experimental
```

Comment:
Since other libraries don’t implement this method and wallets appear not to support them, I’ve moved it to experimental. I’ll eventually remove it if I don’t find a wallet that supports it.